.github/workflows/helm-lint.yaml

```diff
@@ -0,0 +1,41 @@
+name: Reusable workflow for linting helm charts
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        description: "The ref name to checkout the repository."
+        required: false
+        default: ${{ github.event.repository.default_branch }}
+        type: string
+      lint-config-path:
+        description: "The path to the lint configuration file (For an example see https://github.com/helm/chart-testing/blob/main/pkg/config/test_config.yaml)."
+        required: false
+        default: ".github/lint-config.yaml"
+        type: string
+      helm-version:
+        description: "The helm version for setting up poetry."
+        required: false
+        default: "v3.4.0"
+        type: string
+
+jobs:
+  helm-lint:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ inputs.ref }}
+          fetch-depth: 2
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: "${{ inputs.helm-version }}"
```

Comment:
wouldn't it make sense to move the `Check out repository` and the `Set up Helm` step to the composite action of `helm-lint` and add two additional inputs (`ref` and `version`) to it? Since both of these steps will be repeated each time you want to lint helm charts.

Then you will only call one step under the `helm-lint` job:
```yaml
      - name: Lint helm chart
        uses: bakdata/ci-templates/actions/helm-lint@main
        with:
          ref: ${{ inputs.ref }}
          version: version: "${{ inputs.helm-version }}"
          lintConfig: "${{ inputs.lint-config-path }}"
```

Reply:
I don't see a point in having `helm-lint` as a reusable workflow. Because in that case it should be used in as a job. I think it would make sense to have `helm-lint` only as a composite action for using it as one of the steps in a job. WDYT?

Reply:
You are right. I thought composite actions only work inside the same repository or when cloned (which is only true for private repos). In this case we can just call it from the other repo. I also need to change the README.md in the rood directory then

Reply:
If I do not copy the actions into the calling repository anymore, then I encounter a problem with the lint configuration file, because the calling action does not seem to be searching for files inside the calling repository. I will try to find a solution for this issue

Reply:
Nevermind, seems to work now